script.js

```diff
@@ -22,8 +25,8 @@ function generateQuote() {
     .then(function (data) {
       var number = Math.floor(Math.random() * data.length);
       console.log(data[number]);
-      var quoteText = data[number].text;
-      var quoteAuthor = data[number].author;
+      quoteText = data[number].text;
+      quoteAuthor = data[number].author;
```

Comment:
Moved the `quoteText` and `quoteAuthor` variables out of the method call (`then(....)`) and out in the main function, so that it is scoped properly for `shareTweet() ` to be able to read them.